pywbem_mock/_baserepository.py

```diff
@@ -0,0 +1,166 @@
+#
+# (C) Copyright 2020 InovaDevelopment.comn
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+#
+# Author: Karl  Schopmeyer <inovadevelopment.com>
+#
+
+
+"""
+Mock support for the WBEMConnection class to allow pywbem users to test the
+pywbem client without requiring a running WBEM server.
+
+For documentation, see mocksupport.rst.
+"""
+from abc import ABCMeta, abstractmethod
+
+import six
+
+from pywbem import CIMClass, CIMQualifierDeclaration, CIMInstance
+
```

Comment:
Do we want BaseObjStore to be part of the public API? If so, define an `__all__` list containing it.

Reply:
TODO